flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/sql/OverWindowTest.scala

```diff
@@ -44,7 +44,26 @@ class OverWindowTest extends TableTestBase {
       "sum(DISTINCT c) OVER (PARTITION BY b ORDER BY proctime ROWS BETWEEN 2 preceding AND " +
       "CURRENT ROW) as sum2 " +
       "from MyTable"
-
+    val sql2 = "SELECT " +
```

Comment:
I think we can reduce the number of tests is a bit. 
We are basically testing Calcite's parser / validator multiple times with very similar queries. For example, Calcite does not distinguish between proctime and rowtime. One query with a `WINDOW` clause per test case should be sufficient.

Reply:
Agree. I've reduced number of queries per test case (one without `WINDOW ` and one with). Please let me know if it looks good or not